conclave-host/build.gradle

```diff
@@ -13,6 +13,7 @@ dependencies {
     implementation "io.github.classgraph:classgraph:$classgraph_version"
     // Logging using the SLF4J API
     implementation "org.slf4j:slf4j-api:$slf4j_version"
+    implementation "org.tomlj:tomlj:$tomlj_version"
```

Comment:
I need TOML to read the tag that I passed in the Gramine manifest and that needs to be  used by the host